src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -265,14 +265,14 @@
 					x:Name="RunWithPowerShellButton"
 					Width="Auto"
 					MinWidth="40"
-					x:Load="{x:Bind ViewModel.IsPowerShellScript, Mode=OneWay, FallbackValue=False}"
+					x:Load="{x:Bind Commands.RunWithPowershell.IsExecutable, Mode=OneWay}"
 					AutomationProperties.Name="RunWithPowerShell"
-					Command="{x:Bind ViewModel.RunWithPowerShellCommand, Mode=OneWay}"
-					Label="{helpers:ResourceString Name=RunScript}"
+					Command="{x:Bind Commands.RunWithPowershell}"
+					Label="{x:Bind Commands.RunWithPowershell.Label}"
 					LabelPosition="Default"
-					ToolTipService.ToolTip="{helpers:ResourceString Name=RunWithPowerShell}">
+					ToolTipService.ToolTip="{x:Bind Commands.RunWithPowershell.Label}">
```

Comment:
```suggestion
					ToolTipService.ToolTip="{x:Bind Commands.RunWithPowershell.LabelWithHotKey}">
```

Reply:
Which key would you like?

Reply:
We don't need to assign any key, this is just in case the user assigns one.